arborist/auth.go

```diff
@@ -203,17 +262,19 @@ func authorizeUser(request *AuthRequest) (*AuthResponse, error) {
 						WHERE policy.name = ANY($5)
 					)
 				)
-			) FROM usr
-			WHERE usr.name = $1
-		) _, unnest($6::text[])
-		`,
-		request.Username,           // $1
-		request.Service,            // $2
-		request.Method,             // $3
-		len(request.Policies) == 0, // $4
-		pq.Array(request.Policies), // $5
-		pq.Array(resources),        // $6
-	)
+			) _,
+			(SELECT resource.path AS request FROM unnest($6::text[]) JOIN resource ON resource.tag = "unnest") asdf
+			`,
+			request.Username,           // $1
+			request.Service,            // $2
+			request.Method,             // $3
+			len(request.Policies) == 0, // $4
+			pq.Array(request.Policies), // $5
+			pq.Array(resources),        // $6
+		)
+	} else {
+		err = errors.New("missing both resource and tag in auth request")
```

Comment:
I think this error text can be updated to something like `missing resource or tag in auth request`